src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -0,0 +1,224 @@
+    </command>
+    <command source="client" code="0x0002" name="EnableCharging" optional="false" mustUseTimedInvoke="true" apiMaturity="provisional">
+      <access op="invoke" privilege="operate"/>
+      <arg name="ChargingEnabledUntil" type="epoch_s" max="0xFFFFFFFE" isNullable="true"/>
+      <arg name="MinimumChargeCurrent" type="int64s" min="0" max="80000"/>
+      <arg name="MaximumChargeCurrent" type="int64s" min="0" max="80000"/>
+      <description>Allows a client to enable the EVSE to charge an EV.</description>
+    </command>
+    <command source="client" code="0x0003" name="EnableDischarging" optional="true" mustUseTimedInvoke="true" apiMaturity="provisional">
+      <access op="invoke" privilege="operate"/>
+      <arg name="DischargingEnabledUntil" type="epoch_s" max="0xFFFFFFFE" isNullable="true"/>
+      <arg name="MaximumDischargeCurrent" type="int64s" min="0" max="80000"/>
+      <description>Allows a client to enable the EVSE to discharge an EV.</description>
+    </command>
+    <command source="client" code="0x0004" name="StartDiagnostics" optional="true" mustUseTimedInvoke="true" apiMaturity="provisional">
+      <access op="invoke" privilege="operate"/>
+      <description>Allows a client to put the EVSE into a self-diagnostics mode.</description>
+    </command>
+    <command source="client" code="0x0005" name="SetTargets" optional="true" mustUseTimedInvoke="true" apiMaturity="provisional">
+      <access op="invoke" privilege="operate"/>
+      <arg name="DayOfWeekforSequence" type="TargetDayOfWeekBitmap"/>
+      <arg name="ChargingTargets" array="true" type="ChargingTargetStruct"/>
+      <description>Allows a client to set the user specified charging targets.</description>
+    </command>
+    <command source="client" code="0x0006" name="GetTargets" optional="true" response="GetTargetsResponse" mustUseTimedInvoke="true" apiMaturity="provisional">
+      <access op="invoke" privilege="operate"/>
+      <arg name="DaysToReturn" type="TargetDayOfWeekBitmap"/>
+      <description>Allows a client to retrieve the user specified charging targets.</description>
+    </command>
+    <command source="client" code="0x0007" name="ClearTargets" optional="true" mustUseTimedInvoke="true" apiMaturity="provisional">
+      <access op="invoke" privilege="operate"/>
+      <description>Allows a client to clear all stored charging targets.</description>
+    </command>
+    <command source="server" code="0x0000" name="GetTargetsResponse" optional="true" apiMaturity="provisional">
+      <arg name="DayOfWeekforSequence" type="TargetDayOfWeekBitmap"/>
+      <arg name="ChargingTargets" array="true" type="ChargingTargetStruct"/>
+      <description>The GetTargetsResponse is sent in response to the GetTargets Command.</description>
+    </command>
+    <event code="0x0000" name="EVConnected" priority="info" side="server" apiMaturity="provisional">
```

Comment:
There is no "side" for events, right?  All events are server-side....

Reply:
Created new issue : #30665